README.md

```diff
@@ -2,194 +2,118 @@
 [![Build Status](https://travis-ci.org/watson-developer-cloud/conversation-simple.svg?branch=master)](http://travis-ci.org/watson-developer-cloud/conversation-simple)
 [![codecov.io](https://codecov.io/github/watson-developer-cloud/conversation-simple/coverage.svg?branch=master)](https://codecov.io/github/watson-developer-cloud/conversation-simple?branch=master)
 
-This Node.js application demonstrates the Conversation service in a simple chat interface simulating a cognitive car dashboard.
+This Node.js app demonstrates the Conversation service in a simple chat interface simulating a cognitive car dashboard.
 
-To see a running instance of the application demo, [click here](http://conversation-simple.mybluemix.net/).
-
-For more information about the Conversation service, see the [detailed documentation](http://www.ibm.com/watson/developercloud/doc/conversation/).
-
-# Deploying the application
-
-If you want to experiment with modifying the application or use it as a basis for building your own application, you need to deploy it in your own environment. You can then explore the files, make changes, and see how those changes affect the running application. After making modifications, you can deploy your modified version of the application to the Bluemix cloud.
+You can view a [demo][demo_url] of this app.
 
 ## Before you begin
 
-* You must have a Bluemix account, and your account must have available space for at least 1 application and 1 service. To register for a Bluemix account, go to https://console.ng.bluemix.net/registration/. Your Bluemix console shows your available space.
+* Create a Bluemix account
+    * [Sign up][sign_up] in Bluemix, or use an existing account. Your account must have available space for at least 1 app and 1 service.
+* Make sure that you have the following prerequisites installed:
+    * The [Node.js][node_link] runtime (including the [npm][npm_link] package manager)
+    * The [Cloud Foundry][cloud_foundry] command-line client
 
-* You must have the following prerequisites installed:
-  * the [Node.js](http://nodejs.org/) runtime (including the npm package manager)
-  * the [Cloud Foundry command-line client](https://github.com/cloudfoundry/cli#downloads)
+## Installing locally
 
```

Comment:
Because the steps in that section tell you how to install the app, not how to run it. Running is covered by the following section. You have to install before you can run.